docs/4.0/quickstart.md

```diff
@@ -272,7 +272,9 @@ $ tsh --proxy=teleport.example.com join 7645d523-60cb-436d-b732-99c5df14b7c4
 
 We hope this quickstart guide has helped you to quickly set up and play with Teleport. For production environments, we strongly recommend the following:
 
-- Install HTTPS certificates for every Teleport proxy.
+- Install HTTPS certificates for every Teleport proxy. If setting up in a HA config 
+  on AWS, GCP or another cloud you'll likely terminate SSL at a Load Balancer. In 
```

Comment:
terminate `TLS`, `SSL` is a legacy protocol that has been phased out.

Reply:
I've updated this one.